public/angular/userManagement/userManagement.module.js

```diff
@@ -69,10 +74,41 @@ angular.module('tripleT.userManagement', ['ngRoute', 'ngMessages'])
     }
 })
 
+.controller('ChangePwdCtrl', 
+  function($scope, $http, $timeout) {
+    // userInput = {
+    //    password: their current password
+    //    newPassword: what they want it to be
+    //    confirmNewPassword: above. hopefully.   
+    //  }
+    $scope.changePassword = function(userInput) {
+      $scope.passwordChanged = false;
+      $scope.errorMsg = null;
+      if (userInput.newPassword != userInput.confirmNewPassword)
+        $scope.errorMsg = 'New passwords entered do not match';
+      else {
+        $http.post('/auth/changePassword',{
```

Comment:
It's possible to see user passwords. For future: encrypt?

![image](https://cloud.githubusercontent.com/assets/6074188/11165042/471a5a56-8ad1-11e5-960d-732584eef5d5.png)